### ISSUE TITLE
Offer a corrective recommendation when a check has a warning or error

### ISSUE THREAD

**Issue by danielbachhuber** [wp-cli/doctor-command#25]
When a check has a warning or an error, we should offer a corrective recommendation on what they should do to try to address the issue.

**Comment by danielbachhuber** [comment]
For instance, "Files detected with different MIME type" as an error message would be more helpful if it indicated which files had incorrect extensions and how to resolve the error.

**Comment by ibertrix** [comment]
Maybe it can be added an option verbose giving more information?

**Comment by davegaeddert** [comment]
@danielbachhuber do you have any specific ideas about how you'd like to see this implemented? What if another field (`recommendation`, `explanation`, `suggestions` or something) field that was added that isn't enabled by default but can be by using the `--fields` option? I'm guessing some of the text could get pretty long.

**Comment by danielbachhuber** [comment]
> What if another field (recommendation, explanation, suggestions or something) was added that isn't enabled by default but can be by using the `--fields` option?

Yes, this is what I was thinking. We should keep recommendations relatively concise, but we can always link to supplemental documentation.
